sympy/polys/ring_series.py

```diff
@@ -1898,8 +1898,9 @@ def _rs_series(expr, series_rs, a, prec):
     elif expr.is_Mul:
         n = len(args)
         for arg in args:    # XXX Looks redundant
-            R1, _ = sring(arg, expand=False, series=True)
-            R = R.compose(R1)
+            if not arg.is_Number:
```

Comment:
@smichr What is the difference between `is_number` and `is_Number`?
We currently use the `QQ` domain by default, to which `sqrt(2)` does not belong. So we would want control to go inside the `if` block when `arg` is irrational.